src/App.tsx

```diff
@@ -993,7 +995,7 @@ function App(props: RouteComponentProps) {
                         {!isMinimalMode && (
                             <div
                                 style={{
-                                    width: '100%',
+                                    width: `100%`,
```

Comment:
Doesn't seem to be the change needed.

```suggestion
                                    width: '100%',
```